CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [0.5.1
```

Comment:
Missing `]` in `[0.5.1`. You'd also need to update the links at the bottom of this file

Reply:
Typo: "uncessary"